src/i18n/resources/en.js

```diff
@@ -836,13 +836,8 @@ The FRA team fra@fao.org
     totalLandAreaAffectedByFire: 'Total land area affected by fire',
     ofWhichForest: '…of which on forest',
   },
-
   degradedForest: {
     degradedForest: 'Degraded forest',
```

Comment:
Can you also move this ?

Reply:
It is part of https://github.com/openforis/fra-platform/issues/1681

Currently there are references to `sectionName.sectionName` or something similar and would require to refactor everything
To avoid this in this PR, it is left alone in `en.js`